src/EditorFeatures/CSharpTest/Completion/CompletionProviders/ObjectCreationCompletionProviderTests.cs

```diff
@@ -616,5 +616,25 @@ class B { }
 ";
             await VerifyItemExistsAsync(markup, "A");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task NullableTypeCreation()
+        {
+            var markup =
+@"#nullable enable
+namespace ConsoleApplication1
+{
+    class Program
+    {
+        void M()
+        {
+            object? o;
+            o = new $$
+        }
+    }
+}
+";
+            await VerifyItemExistsAsync(markup, "object");
```

Comment:
can we have test that shows this does the right thing:  `List<object?> list = new $$`.

Reply:
I'm almost always up for adding more tests :)